main.go

```diff
@@ -67,6 +68,16 @@ func main() {
 
 	// 如果启用了Unix
 	if conf.UnixConfig.Listen != "" {
+		// delete socket file before listening
+		if _, err := os.Stat(conf.UnixConfig.Listen); err == nil {
+			util.Log().Info("检测到 socket 文件已存在，删除中...")
```

Comment:
We can remove this line to make log less verbose, same as line 78

Reply:
resolved in the latest update